### PR TITLE
Add rak enclosure harbor breeze solar hack how-to

### PR DESCRIPTION
**Pull request by tavdog**
Add a link to the RAK enclosures page and also add a HOW-TO page under RAK dir.

### CODE REVIEW

**Review by pdxlocations — CHANGES_REQUESTED** [review]
Awesome project!  Could you move the build into a sub-page of enclosures per @rcarteraz and @tropho23 's request on discord?

**Comment by pdxlocations** [comment]
Your preview Vercel link from Discord helps with tracking the changes:
https://meshtastic-docs-sigma.vercel.app/docs/hardware/devices/rak/harbor_breeze_solar

**Comment by tavdog** [comment]
> Awesome project! Could you move the build into a sub-page of enclosures per @rcarteraz and @tropho23 's request on discord?

I would but I don't know how to create a sub folder with this markup.

**Comment by pdxlocations** [comment]
> > Awesome project! Could you move the build into a sub-page of enclosures per @rcarteraz and @tropho23 's request on discord?
> 
> I would but I don't know how to create a sub folder with this markup.

- You'll want to create a new folder /docs/hardware/devices/rak/enclosures
- move enclosures.mdx into the new folder
- rename enclosures.mdx to index.mdx
- move harbor_breeze_solar_enclosure.mdx into the new folder
- change your sidebar_position to 1

**Comment by pdxlocations** [comment]
One other thing, are you able to reduce the file sizes of your images?  They load really slow.  

Screenshot 2024-01-21 at 10 05 04 AM

File: docs/hardware/devices/rak/enclosures.mdx [hunk 1]
```diff
@@ -38,3 +38,10 @@ Download from [Printables](https://www.printables.com/model/286662-rak19003-case
 Download from [Printables](https://www.printables.com/model/286664-rak19003-micro-case-for-meshtastic) or purchase from the creator's [Etsy Store](https://www.etsy.com/listing/1302260756/meshtastic-rak19003-micro-case).
 
 ![rak19003-micro](/img/enclosures/3dp-tropho-rak19003-micro.png)
+
+## Created by tavdog/Tavis Gustafson
+
+### [Harbor Breeze Meshtastic Hack](./harbor_breeze_solar_enclosure.mdx)
+
+![Harbor Breeze Solar LED Meshtastci Hack Complete](/img/enclosures/hbmh_complete.png)
```
Comment (pdxlocations) [inline docs/hardware/devices/rak/enclosures.mdx:46]: typo here "Meshtastci"

**Comment by tavdog** [comment]
> > > Awesome project! Could you move the build into a sub-page of enclosures per @rcarteraz and @tropho23 's request on discord?
> > 
> > 
> > I would but I don't know how to create a sub folder with this markup.
> 
> * You'll want to create a new folder /docs/hardware/devices/rak/enclosures
> * move enclosures.mdx into the new folder
> * rename enclosures.mdx to index.mdx
> * move harbor_breeze_solar_enclosure.mdx into the new folder
> * change your sidebar_position to 1

Ok. i'll give a try and reduce image size.
and fix mastastcli ( just alt image text though )

**Review by pdxlocations — APPROVED** [review]

**Pull request merged**